framework/src/modules/dpos_v2/utils.ts

```diff
@@ -234,6 +234,24 @@ export const isPunished = (
 	);
 };
 
+export const lastHeightOfRound = (height: number, genesisHeight: number, roundLength: number) => {
+	const roundNumber = Math.ceil((height - genesisHeight) / roundLength);
+
+	return roundNumber * roundLength + genesisHeight;
+};
+
+export const isCertificateGenerated = (options: {
```

Comment:
maybe unit test for this func

Reply:
Added a test in the `unlock.spec` file which tests the utility function